assets/scss/_card.scss

```diff
@@ -8,12 +8,13 @@
 }
 .d-person-card {
   padding: 0 !important;
+  width: 300px;
```

Comment:
instead of setting the width here, can a different width variant be passed to `DCard`?